COMPLETE LIST OF SCRIPTS.vbs

```diff
@@ -2493,7 +2493,7 @@ script_array(script_num).subcategory            = array("HEALTH CARE")
 script_array(script_num).release_date           = #10/01/2000#
 script_array(script_num).hot_topic_link			= ""
 script_array(script_num).used_for_elig			= False
-script_array(script_num).policy_references		= array("")						'SEE Line 58 for format'
+script_array(script_num).policy_references		= array("EPM Asset_Assessment_for_Planning_Purposes http://hcopub.dhs.state.mn.us/epm/2_4_2_1_1.htm", "ONESOURCE Processing_an_Asset_or_Income_Transfer_for_MA-LTC https://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-170126")						'SEE Line 58 for format'
```

Comment:
I think that this EPM reference hits the mark a little closer than the one you found. [2.4.2.1.2 MA-LTC Community Spouse Asset Allowance (state.mn.us)](http://hcopub.dhs.state.mn.us/epm/2_4_2_1_2.htm?rhhlterm=12%20community%20spouse&rhsearch=12%20months%20community%20spouse). 

Also this took me FOREVER to find, and this is a policy that I know.

Reply:
Additionally I didn't find any other references to add in POLI TEMP or any where else.

Reply:
Awesome- I've swapped the link out in both locations.